src/app/core/components/admin/interfaces/course.interface.ts

```diff
@@ -0,0 +1,6 @@
+export class ICourse {
+  id?: string;
+  label: string;
+  description: string;
```

Comment:
Contact with BE developer, to confirm that description is required

Reply:
be said that its required for now